vcr/interface.go

```diff
@@ -68,6 +69,18 @@ type Writer interface {
 	StoreRevocation(r credential.Revocation) error
 }
 
+// TrustManager bundles all trust related methods in one interface
+type TrustManager interface {
+	// Trust adds trust for a Issuer/CredentialType combination. The added trust is persisted to disk.
```

Comment:
`persisted to disk.` implementation detail that shouldn't be part of godoc (since it might be an implementation that doesn't persist to disk, but to somewhere else or not at all, who knows).